static/app/components/group/groupSummary.tsx

```diff
@@ -0,0 +1,141 @@
+import styled from '@emotion/styled';
+
+import FeatureBadge from 'sentry/components/badge/featureBadge';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import Panel from 'sentry/components/panels/panel';
+import * as SidebarSection from 'sentry/components/sidebarSection';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Group} from 'sentry/types/group';
+import marked from 'sentry/utils/marked';
+import {type ApiQueryKey, useApiQuery} from 'sentry/utils/queryClient';
+
+interface GroupSummaryProps {
+  group: Group;
+}
+
+interface GroupSummaryData {
+  group_id: string;
+  impact: string;
+  summary: string;
+}
+
+const makeGroupSummaryQueryKey = (groupId: string): ApiQueryKey => [
+  `/issues/${groupId}/summarize/`,
```

Comment:
this should still be the full url since hybrid cloud is off on self hosted 
```suggestion
  `/organizations/${organizationSlug}/issues/${groupId}/summarize/`,
```